chainguard-dev/edu
# removing Discovery onboarding doc and references to Agentless

## Type of change

This PR removes our Discovery Onboarding doc and references to "Agentless" connections from the Enforce Docs.

### What should this PR do?

resolves #1178 

### Why are we making this change?

This comes from a request from @tcnghia (outlined [here](https://docs.google.com/document/d/11gT2pwfasvzIQbtg0GQqTGKL0ojhexNrEi7gUm863k4/edit?disco=AAABAHy9r68))

### What are the acceptance criteria? 



### How should this PR be tested?

The goal was to remove any references to "agentless" (as determined from [this search](https://www.google.com/url?q=https://github.com/search?q%3Drepo%253Achainguard-dev%252Fedu%2520agentless%26type%3Dcode&sa=D&source=docs&ust=1701284909254443&usg=AOvVaw1TopxulKeryC6vorC2ATjw)). I don't think removing/changing this content has broken any links or anything, but I guess just give it a once over to see that I didn't miss any other references to agentless/discovery.
